website/docs/reference/commands/clone.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "About dbt clone command"
+sidebar_label: "clone"
+id: "clone"
+---
+
+The `dbt clone` command clones selected nodes from the [specified state](/reference/node-selection/syntax#establishing-state) to the target schema(s). This command makes use of the `clone` materialization:
+- If your data platform supports zero-copy cloning of tables, and this model exists as a table in the source environment, dbt will create it in your target environment as a clone
+- Otherwise, dbt will create a simple pointer view (`select * from` the source object)
+- By default, `dbt clone` will not recreate pre-existing relations in the current target. To override this, use the `--full-refresh` flag. 
+- You may want to specify a higher number of [threads](/docs/running-a-dbt-project/using-threads) to decrease execution time since individual clone statements are independent of one another.
+
+The `clone` command is useful for:
+- blue/green continuous deployment
```

Comment:
yes, good call! that's a fairly old post at this point, but until we have a newer one, it's good to provide some definition/context

Reply:
How do we feel about this being a snowflake-specific article? I think blue/green continuous blue/green deployment will be different depending on the warehouse being used (zero copy cloning vs. pointer views)

Reply:
Even better point. Blue/green is really only enabled by table clones. There's also ~hackery~ cleverness in that post around overriding `ref` so that when the schemas get swapped, the views stay pointed to the same database.

Maybe better to skip it for now, and just say something like:
> - blue/green continuous deployment (on data warehouses that support cloning tables)

Or remove the bullet entirely, until we have a more up-to-date and thorough dev blog article that we could link to

Reply:
Just updated - let me know what you think